sql/migrations/20210515110157_world.sql

```diff
@@ -0,0 +1,851 @@
+(576, 2),
+(584, 2),
+(608, 2),
+(627, 2),
+(753, 2),
+(756, 2),
+(792, 2),
+(809, 2),
+(810, 2),
+(849, 2),
+(864, 2),
+(911, 2),
+(912, 2),
+(936, 2),
+(951, 2),
+(962, 2);
+
+-- Tanaris Winter Reveler Correct Position
+UPDATE `creature` SET `position_x` = -7160.19, `position_y` = -3845.04, `position_z` = 8.8656, `orientation` = 2.89725 WHERE `guid` = 90180;
+
+-- Auberdine Winter Reveler Correct Position
+UPDATE `creature` SET `position_x` = 6406.53, `position_y` = 519.009, `position_z` = 8.73281, `orientation` = 6.07375 WHERE `guid` = 90187;
+UPDATE `creature` SET `orientation` = 2.84489 WHERE `guid` = 90188;
+
+-- Booty Bay Winter Reveler Correct Position
+UPDATE `creature` SET `position_x` = -14460.8, `position_y` = 492.998, `position_z` = 15.208, `orientation` = 3.92699 WHERE `guid` = 89380;
+
+-- Correct Scale for Tauren Female Winter Reveler
+UPDATE `creature_template` SET `display_scale1` = 1.25 WHERE `entry` = 15789;
+
+-- Correct Scale for Tauren Male Winter Reveler
+UPDATE `creature_template` SET `display_scale1` = 1.35 WHERE `entry` = 15793;
+
+-- Greatfather Winter's Helper Should Yell
+DELETE FROM `creature_ai_scripts` WHERE `id` = 1574601;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (1574601, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 11430, 0, 0, 0, 0, 0, 0, 0, 0, 'Great-father Winter\'s Helper - Say Text');
+
+-- Quest 6962 Is Not Repeatable
+UPDATE `quest_template` SET `SpecialFlags` = 0 WHERE `entry` = 6962;
+
+-- Falling underground - Fixes (credit cmangos)
+UPDATE creature SET position_x = -6690.331543, position_y = -2988.697021, position_z = 248.367706, wander_distance = 7, movement_type = 1 WHERE guid = 6910;
+UPDATE creature SET position_x = -6629.925293, position_y = -3554.510498, position_z = 254.083786, wander_distance = 7, movement_type = 1 WHERE guid = 6926;
+UPDATE creature SET position_x = 51.213818, position_y = 1539.047974, position_z = 107.961784, wander_distance = 7, movement_type = 1 WHERE guid = 27980;
+UPDATE creature SET position_x = 393.519775, position_y = -623.946899, position_z = 162.409653, wander_distance = 5, movement_type = 1 WHERE guid = 17345;
+UPDATE creature SET position_x = -3649.268, position_y = -727.9708, position_z = 10.7234, orientation = 5.907869, wander_distance = 0, movement_type = 0 WHERE guid = 9530;
+
+-- Elder Shadowmaw Panther 1713 (credit cmangos)
+-- swimming anim fixed
+DELETE FROM creature_addon WHERE guid IN (SELECT guid FROM creature WHERE id = 1713);
+UPDATE creature_template_addon SET bytes1 = 0 WHERE entry = 1713;
+
+-- Ridge Stalker 2731 (credit cmangos)
+-- swimming anim fixed
+DELETE FROM creature_addon WHERE guid IN (SELECT guid FROM creature WHERE id = 2731);
+UPDATE creature_template_addon SET bytes1 = 0, auras = NULL WHERE entry = 2731;

```

Comment:
This table does not exist. Same for above creature. Don't copy paste things from cmangos without testing!